contrib/net/http/http_test.go

```diff
@@ -516,11 +516,11 @@ func router(muxOpts ...Option) http.Handler {
 	return mux
 }
 
-func handler200(w http.ResponseWriter, _ *http.Request) {
+func handler200(w http.ResponseWriter, r *http.Request) {
```

Comment:
In the body of the function this variable isn't used. Is it needed due to another reason?